From 78e5e6f42d4f842dca6788d0aa5edd0a6d466ccc Mon Sep 17 00:00:00 2001
From: Fugang Duan <fugang.duan@nxp.com>
Date: Mon, 24 Jul 2017 11:38:48 +0800
Subject: [PATCH 2188/5242] MLK-16064 arm64: dts: imx8qxp: force enet1 use PHY
 clock delay on lpddr4 arm2 board

commit  441d8ecefa716016fbd5ea4d7f534f1f0a5c64d6 from
https://source.codeaurora.org/external/imx/linux-imx.git

Currently enet1 use MAC clock delay, there have packet error
in 100Mbps mode, no packet error in Gbps mode. Still use PHY
clock delay instead of MAC that 100Mbps mode has better timing
and no frame error.

Signed-off-by: Fugang Duan <fugang.duan@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../boot/dts/freescale/fsl-imx8qxp-lpddr4-arm2.dts |    4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-lpddr4-arm2.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-lpddr4-arm2.dts
index 33a3ce9..2f14584 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-lpddr4-arm2.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-lpddr4-arm2.dts
@@ -373,11 +373,9 @@
 &fec1 {
 	pinctrl-names = "default";
 	pinctrl-0 = <&pinctrl_fec1>;
-	phy-mode = "rgmii";
+	phy-mode = "rgmii-id";
 	phy-handle = <&ethphy0>;
 	fsl,magic-packet;
-	fsl,rgmii_txc_dly;
-	fsl,rgmii_rxc_dly;
 	status = "okay";
 
 	mdio {
-- 
1.7.9.5

